javascript/operators/destructuring.json

```
@@ -144,10 +144,10 @@
                 "version_added": "36"
               },
               "safari": {
-                "version_added": false
+                "version_added": "9"
```

```suggestion
                "version_added": "9.1"
```
It looks like https://trac.webkit.org/changeset/192671/webkit, 19 Nov 2015, was the revision which landed support for rest parameters. I think that means it couldn’t have gone into Safari 9, which shipped on 30 Sep 2015, but instead went into Safari 9.1, which shipped on 21 Mar 2016.

---

Agh, I forgot: Sauce Labs' "Safari 9" is actually "Safari 9.1"...